scrapers/fifty_a/fifty_a/spiders/officer.py

```diff
@@ -18,19 +18,37 @@ class OfficerSpider(CrawlSpider):
     )
 
     def parse_officer(self, response):
-        race, gender = self.parse_race_and_gender(response)
+        _, gender = self.parse_race_and_gender(response)
         officer = OfficerItem(
+            taxnum=self.parse_taxnum(response),
             url=response.url,
-            name=response.css("h1::text").get(),
-            badge=response.css(".badge::text").get(),
-            race=race,
             gender=gender,
             complaints=self.parse_complaints(response),
-            age=response.css(".age::text").get(),
+            age=self.parse_age(response),
         )
 
         yield officer
 
+    def parse_age(self, response):
+        age = response.css(".age::text").get()
+        if age:
+            age = parse_string_to_number(age)
```

Comment:
In some cases, this value returns a range. That's leading to a crash in the parser and the loss of the entry.
```
  File "/police-data-trust-scrapers/scrapers/fifty_a/fifty_a/spiders/officer.py", line 35, in parse_age
    age = parse_string_to_number(age)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/police-data-trust-scrapers/scrapers/common/parse.py", line 12, in parse_string_to_number
    raise NotImplementedError(e)
NotImplementedError: invalid literal for int() with base 10: '65-69'
```

Reply:
I can look into this as well, I did not realize this returned a range. Thanks for bringing it to my attention. In this scenario if it returned a range, would it be better to take some valid number in the range(i.e lowest, highest, average, median, etc.) or would it be better to instead change the type to string and keep the range as is or perhaps something else entirely?

Reply:
I think the best move is to take the midpoint of the range. If it's an estimate anyway, that should be fine.